moveit_core/collision_detection/include/moveit/collision_detection/collision_plugin.h

```diff
@@ -66,9 +66,9 @@ MOVEIT_CLASS_FORWARD(CollisionPlugin)  // Defines CollisionPluginPtr, ConstPtr,
  *   class MyCollisionDetectionLoader : public CollisionPlugin
  *   {
  *   public:
- *     virtual bool initialize(const planning_scene::PlanningScenePtr& scene, bool exclusive) const
+ *     virtual bool initialize(const planning_scene::PlanningScenePtr& scene) const
  *     {
- *       scene->setActiveCollisionDetector(my_collision_checker::MyCollisionDetectorAllocator::create(), exclusive);
+ *       scene->setActiveCollisionDetector(my_collision_checker::MyCollisionDetectorAllocator::create());
```

Comment:
Good cleanup! We should also rename all `setActiveCollisionDetector` references to `allocateCollisionDetector` per #464